systems/x86_64-linux/boba-social/default.nix

```diff
@@ -1,17 +1,33 @@
-{ pkgs, ... }: {
+{ pkgs, inputs,  ... }: {
   imports = [
     ./hardware.nix
+    inputs.vscode-server.nixosModules.default
```

Comment:
Even better, we can put this in flake.nix and let Snowfall Lib take care of importing it.

```nix
systems.modules = with inputs; [
  vscode-server.nixosModules.default
];
```